version/version.go

```diff
@@ -17,5 +17,5 @@ func FormattedVersion() string {
 	if Version == "" {
 		return "unknown"
 	}
-	return fmt.Sprintf("%s (%s @ %s)\n", Version, BuildVersion, BuildTime)
+	return fmt.Sprintf("%s (%s @ %s)", Version, BuildVersion, BuildTime)
```

Comment:
Just want to confirm this didn't accidentally _remove_ a trailing newline from `--version` right?